starcluster/plugins/efs.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2015 Michael Cariaso
+#
+# This file is a plugin for StarCluster.
+#
+# This plugin is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# StarCluster is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with StarCluster. If not, see <http://www.gnu.org/licenses/>.
+
+from starcluster import clustersetup
+from starcluster.logger import log
+import boto3
+
+
+class EFSPlugin(clustersetup.DefaultClusterSetup):
+    """Add EFS support
+
+    Example config:
+
+    [plugin efs]
+    SETUP_CLASS = efs.EFSPlugin
```

Comment:
You may have an explicit plugin path somewhere, so that's possible. I was using it on a fairly vanilla setup. I'm planning to test it out some more and will post any issues I have.